README.md

```diff
@@ -15,6 +15,28 @@ Revisit your solution to homework #1, problem 5, which expresses an improper fra
 **a)** Write a function `fractionString(n,d)` which takes 2 parameters (n,d) and returns a string.  For example, 
 `fractionString(7,4)` should return "1 3/4", and `fractionString(3,3)` should (for now) return "1 0/3".  As before, assume that _n_ and _d_ are both positive integers.
 
+//Solution 1a
+
+function fractionString (n,d); {
+	var n = 7;
+    var d = 4;
+
+    var output = "";
+
+    var w = Math.floor(n/d);
+
+    if( w > 0) {
+        output += w + " ";
+    }
+
+    var nn = n % d;
+
+    if(nn > 0) {
+        output += nn + "/" + d;
+    }
+
+    console.log(output);
+
```

Comment:
Great Job. this Technically works but on lines 21 and 22 you are "hard coding" the values of n and d. These values are evaluated correctly, but since you are hard coding the values of n and d, you are overwriting any arguments being passed into the function. Therefore console.log(fractionString(3,4)); will output 1 3/4 not just 3/4.